src/install.rst

```diff
@@ -36,7 +36,19 @@ When you use Docker to run Nextstrain components, you don’t need to manage any
 Installation Steps
 ==================
 
-These instructions will install the Nextstrain CLI and tools to run and view your own Nextstrain analyses. Configuration options vary by operating system.
+These instructions will install the Nextstrain CLI and tools to run and view your own Nextstrain analyses. Configuration options vary by platform.
+
+.. note::
+
+   For each operating system, there are two options:
+
+   1. **Docker**: use if you plan to use Nextstrain without other bioinformatics tools. Docker provides the most predictable environment.
+   2. **Native**: use if you plan to use other bioinformatics tools alongside Nextstrain. Native installation is done via conda, allowing flexibility in managing your own conda environments.
```

Comment:
Hmm. I think I know what you intend here, but as it's worded, this reads as potentially misleading to me. I think, but correct me if wrong, that the intent is to communicate that if you want to use other arbitrary bioinformatics tools _within a `nextstrain build` run_ then it'll be (much) easier to do that with "native" installation via Conda. That emphasized bit of scoping seems important to me, as there's nothing about either installation method that makes it harder or easier to use other bioinformatics tools on Nextstrain inputs before hand, outputs afterwards, etc.

It might also be worth mentioning in this note that _both_ options can be followed, if you know up-front you want the flexibility to switch between then or (more realistically perhaps) you realize later you initially choose the wrong option for your needs.

Relatedly, ISTM that we're going to say "use Conda if you want to use other bioinformatics tools in your `nextstrain build` run", then we should also probably include some information about how to actually do that (i.e. how to install other arbitrary things into the Conda env). This implies to me that a separate doc page might be in order, something like a how-to _Use Nextstrain with other bioinformatics tools_.

Reply:
I agree with @tsibley about the potential for confusion. Some of this choice comes down to whether users want to use Docker/Conda or not. We've heard from folks at office hours who do not want to use Docker (or Conda), and they will choose the installation platform that they do prefer. For people who don't know what they prefer, it still makes sense to me to recommend Docker (and flag Docker as the recommended approach).

Other components of this choice that seem most relevant are well-described here:

  - "Docker provides the most predictable environment"
  -  managing your own conda environments

In the past we've talked about this kind of decision as "if you know what you're doing, here are your options". The same context applies to the Windows vs. Windows/WSL choice.

We should discuss this more as a small group and refine this PR based on our discussion...

Reply:
I agree with @tsibley about the potential for confusion. Some of this choice comes down to whether users want to use Docker/Conda or not. We've heard from folks at office hours who do not want to use Docker (or Conda), and they will choose the installation platform that they do prefer. For people who don't know what they prefer, it still makes sense to me to recommend Docker (and flag Docker as the recommended approach).

Other components of this choice that seem most relevant are well-described here:

  - "Docker provides the most predictable environment"
  -  managing your own conda environments

In the past we've talked about this kind of decision as "if you know what you're doing, here are your options". The same context applies to the Windows vs. Windows/WSL choice.

We should discuss this more as a small group and refine this PR based on our discussion...

Reply:
> We should discuss this more as a small group and refine this PR based on our discussion...

+1